src/dlgRoomProperties.cpp

```diff
@@ -0,0 +1,555 @@
+        lineEdit_roomSymbol->setText(mpSymbols.keys().first());
+        comboBox_roomSymbol->hide();
+    } else {
+        // show combined dropdown & text-entry box to host all of the (multiple) existing symbols
+        lineEdit_roomSymbol->hide();
+        comboBox_roomSymbol->addItems(getComboBoxSymbolItems());
+    }
+    initSymbolInstructions();
+
+    // Configure icon display
+    auto pFirstRoom = *(pRooms.begin());
+    selectedSymbolColor = pFirstRoom->mSymbolColor;
+    if (pColors.size() == 1) {
+        mRoomColor = mpHost->mpMap->getColor(pFirstRoom->getId());
+        mRoomColorNumber = pColors.keys().first();
+    } else {
+        mRoomColor = QColor("grey"); // rgb(128, 128, 128)
+        mRoomColorNumber = -1;
+    }
+    slot_updatePreview();
+
+    //   Configure weight display
+    mpWeights = pWeights;
+    if (mpWeights.isEmpty()) {
+        // show spin-box with default value
+        spinBox_weight->setValue(1);
+        comboBox_weight->hide();
+    } else if (mpWeights.size() == 1) {
+        // show spin-box with the (single) existing weight pre-filled
+        spinBox_weight->setValue(mpWeights.keys().first());
+        comboBox_weight->hide();
+    } else {
+        // show combined dropdown & text-entry box to host all of the (multiple) existing weights
+        spinBox_weight->hide();
+        comboBox_weight->addItems(getComboBoxWeightItems());
+    }
+    initWeightInstructions();
+
+    // Configure lock display
+    // Are all locks the same or mixed status? Then show dialog in tristate.
+    if (lockStatus.contains(true) && lockStatus.contains(false)) {
+        checkBox_locked->setTristate(true);
+        checkBox_locked->setCheckState(Qt::PartiallyChecked);
+    } else if (lockStatus.contains(true)) {
+        checkBox_locked->setTristate(false);
+        checkBox_locked->setCheckState(Qt::Checked);
+    } else { // lockStatus.contains(false)
+        checkBox_locked->setTristate(false);
+        checkBox_locked->setCheckState(Qt::Unchecked);
+    }
+    initLockInstructions();
+
+    // Configure dialog display
+    adjustSize();
+}
+
+
+void dlgRoomProperties::initLockInstructions()
+{
+    QString instructions = tr("Lock room(s), so it/they will never be used for speedwalking",
+                           // Intentional comment to separate arguments!
+                           "This text will be shown at a checkbox, where you can set/unset a number of room's lock.",
+                           mpRooms.size());
+    checkBox_locked->setText(instructions);
+    /* checkBox_locked->setWordWrap(true); // Unfortunately, this does not exist, yet. */
+}
+
+
+void dlgRoomProperties::initWeightInstructions()
+{
+    if (mpWeights.empty()) {
+        label_weightInstructions->hide();
+        return;
+    }
+
+    QString instructions;
+    if (mpWeights.size() == 1) {
+        instructions = tr("Enter a new room weight to use as the travel time for all of the %n selected room(s). "
```

Comment:
@SlySven we can fix the room(s) to be just 'room' with the en_US translations, right?

Reply:
Yes the final dialog will show room or rooms respectively but no brackets anymore.